www/topical-software.rst

```diff
@@ -472,7 +472,7 @@ Miscellaneous
 - `xarray <http://xarray.pydata.org/en/stable/>`__: Library that allows the labeling of any dimension in a multi-dimensional array.
 - `PyCVF <http://pycvf.sourceforge.net>`__: A computer vision and videomining Framework.
 
-- `CNEMLIB <https://sn-m2p.cnrs.fr/SphinxDoc/cnem/index.html>`__ : propose an implementation of CNEM in 2d and 3d. The CNEM is a generalisation for non convex domain of the Natural Element Method. It's a FEM like approach. The main functionalities of CNEMLIB are : i) interpolation of scattered data spread on convex or non convex domains with the Natural Neighbour interpolant (Sibson) in 2d, and the Natural Neighbour interpolant (Sibson or Laplace) or the linear finite element interpolant over the Delaunay tessellation in 3d. ii) a gradient matrix operator which allows to calculate nodal gradients for scattered data (the approach used is based on the stabilized nodal integration, SCNI). iii) a general assembling tools to construct assembled matrix associated with a weak formulation (heat problem, mechanic problem, hydrodynamic problem, general purpose problem) as such used with the Finite Element Method (FEM).
+- `CNEMLIB <https://m2p.cnrs.fr/sphinxdocs/cnem/cnem3d.html>`__ : propose an implementation of CNEM in 2d and 3d. The CNEM is a generalisation for non convex domain of the Natural Element Method. It's a FEM like approach. The main functionalities of CNEMLIB are : i) interpolation of scattered data spread on convex or non convex domains with the Natural Neighbour interpolant (Sibson) in 2d, and the Natural Neighbour interpolant (Sibson or Laplace) or the linear finite element interpolant over the Delaunay tessellation in 3d. ii) a gradient matrix operator which allows to calculate nodal gradients for scattered data (the approach used is based on the stabilized nodal integration, SCNI). iii) a general assembling tools to construct assembled matrix associated with a weak formulation (heat problem, mechanic problem, hydrodynamic problem, general purpose problem) as such used with the Finite Element Method (FEM).
```

Comment:
The home page seems to be https://m2p.cnrs.fr/sphinxdocs/cnem/, not https://m2p.cnrs.fr/sphinxdocs/cnem/cnem3d.html

Reply:
true, thanks for the remark